content/blog/2015-06-18-practical-anomaly-detection.md

```diff
@@ -83,8 +83,6 @@ ALERT InstanceLatencyOutlier
       >
         1
   FOR 30m
-  SUMMARY "{{$labels.instance}} in {{$labels.job}} is a latency outlier"
```

Comment:
Don't wanna keep at least one of these in the new format? To showcase Prometheus's alert message templating on the side a bit?